app/views/shared/_navbar.html.erb

```diff
@@ -2,7 +2,7 @@
   <div class="container-fluid">
     <!-- Brand and toggle get grouped for better mobile display -->
     <div class="navbar-header">
-      <a class="navbar-brand" href="#">CloneBnB</a>
+      <a class="navbar-brand" href=" <%= listings_path %> ">CloneBnB</a>
```

Comment:
glad this is added in. I think this should eventually go to the root, when we do some queries to find the featured listings